modules/UI/videolayout/VideoLayout.js

```diff
@@ -423,7 +423,9 @@ var VideoLayout = {
      * Shows the presence status message for the given video.
      */
     setPresenceStatus (id, statusMsg) {
-        remoteVideos[id].setPresenceStatus(statusMsg);
+        let remoteVideo = remoteVideos[id];
```

Comment:
Not a big deal, but maybe we could add a getter for getting remote video instead of duplicating direct access everywhere ?